src/parquet/util/rle-encoding.h

```diff
@@ -26,6 +26,7 @@
 #include "parquet/util/compiler-util.h"
```

Comment:
We are available to help, if help is needed. Indeed, https://github.com/lemire/simdcomp is solid C library that can deliver multiple time the performance of a scalar implementation by making explicit use of SIMD instructions. It is possible to achieve good gains over a scalar Parquet decoder using deliberate SIMD code. I am personally available to look at this if there is interest...